TEveEventDisplay/src/TEveEventDisplay_module.cc

```diff
@@ -56,13 +58,15 @@ namespace mu2e
       int _diagLevel;
       bool _showBuilding;
       bool _showDSOnly;
+      bool _showInsidePS;
       bool _showCRV;
       bool _showEvent;
```

Comment:
In parallel to the change I requested in fcl, please put  all of the "_show*" member data into a struct.  This will reduce maintenance at the function calls as this struct becomes more complex.  Remember to pass the struct by const reference.  You should nest the configuration code for the struct; ie move it out of the top level and into the code for the struct.

Reply:
Sure, I can do that, I have draw options in a struct now too